js/donations-visualiser.js

```diff
@@ -855,20 +855,49 @@ function update(partyNodes, parties, selectedParties, resetControls) {
 
     d3.select("#party_select").selectAll(".checkbox").remove();
 
-    party_checkboxes = d3.select("#party_select").selectAll(".checkbox")
+    parties_grouped = []
+    parties_grouped_search = ["Coalition", "Australian Labor Party", "Australian Greens", "Palmer United Party"]
+    parties.map(function(item){
```

Comment:
In fact, you can skip doing another pass and filter out the minor parties here too...
```
var minor_parties = [];
var parties_grouped = parties.reduce(function(accumulator, party) {
  var index = parties_grouped_search.indexOf(party_map[party].name);
  if (index > -1) {
    accumulator[index] = party;
  } else {
    minor_parties[index] = party;
  }
  return accumulator;
}, []);
```

Then you can remove the filter on line 890.